samples/snippets/csharp/new-in-7/new-in-7/throwexpressions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace new_in_7
+{
+    public class ConfigResource
+    {
+        
+    }
+
+    public class ApplicationnOptions
+    {
+        private string name;
+
+        #region 38_ThrowInInitialization
+        private ConfigResource loadedConfig = LoadConfigResourceOrDefault() ?? 
+            throw new InvalidOperationException("Could not load config");
+        #endregion
+
+        #region 39_ThrowInConstructor
+        public ApplicationnOptions()
```

Comment:
@tverboon Thanks for spotting that.  Since this PR had already been merged, I created a new PR to fix the issue: #1361